ledger/catchpointtracker.go

```diff
@@ -1593,16 +1593,52 @@ func (ct *catchpointTracker) accountsInitializeHashes(ctx context.Context, tx *s
 		// if anything goes wrong, it will still get rolled back.
 		_, err = trie.Evict(true)
 		if err != nil {
-			return fmt.Errorf("accountsInitialize was unable to commit changes to trie: %v", err)
+			return fmt.Errorf("initializeHashes was unable to commit changes to trie: %v", err)
+		}
+
+		// Now add the kvstore hashes
+		// CR Question: Does this neeed ordering? I don't see why it would, but
+		// the account iterator is explicitly ordered for some reason.
+		kvs, err := tx.QueryContext(ctx, "SELECT key, value FROM kvstore")
```

Comment:
makeOrderedAccountsIter reads ordered, shouldn't this do the same? IDK

Reply:
See the discussion in slack. @cce tracked down why the accounts were ordered - it had to do with ordering the _hashes_ so that it was faster to insert them into the trie. But, given that this code only runs one time, in an uncommon situation (a node was running without catchpoints, then turns them on) I definitely don't think it's worth the effort and complexity to create a temp table, then grab the hashes in order, just to build the tree faster.